ATTRIBUTIONS/wasabi-experiment-objects-attribution.xml

```diff
@@ -535,7 +535,7 @@
          <type>jar</type>
          <licenses/>
          <downloadUrls>
-            <string>TBD</string>
+            <string>file:/Users/jtodd1/Documents/projects/wasabi/modules/experiment-objects/../../repository/Hyrule/Hyrule/0.0.1/Hyrule-0.0.1.jar</string>
```

Comment:
ahhhhh ... we need to run [ws]/bin/ATTRIBUTIONS_patch.sh

this is a stop-gap measure until we remove the local file repository, necessary now until we release hyrule source code.

i just pushed up a revision post filter.